tasks/upload.py

```diff
@@ -215,6 +215,13 @@ def _should_debounce_processing(upload_context: UploadContext) -> Optional[float
     return None
 
 
+class CreateUploadResponse(TypedDict):
```

Comment:
nice nice

Reply:
How's the naming here? I was not _amazed_ by it 😂